web/web-frontend/src/components/project/rest/method/modal/UpdateMethodModal.js

```diff
@@ -129,9 +140,11 @@ class UpdateMethodModal extends PureComponent {
     }
 
     onDefaultMockResponseIdChange(defaultMockResponseId){
+        console.log(defaultMockResponseId)
         this.setState({ updateMethod: {
                 ...this.state.updateMethod,
-                defaultMockResponseId: defaultMockResponseId
+                defaultMockResponseId: defaultMockResponseId,
+                automaticForward: defaultMockResponseId == "-- select an option --" ? this.state.updateMethod.automaticForward : false
```

Comment:
Again here we make sure that if the user selects a default mock response then we automatically disable the automatic forward since we would always fallback to this default response if nothing is matching the request.